test/membrane/rtcp/twcc_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Membrane.RTCP.TWCCTest do
+  use ExUnit.Case, async: true
+
+  alias Membrane.RTCP.Fixtures
+  alias Membrane.RTCP.TransportFeedbackPacket.TWCC
+
+  describe "TWCC module" do
+    test "encodes and decodes valid feedbacks" do
+      encoded_feedbacks = Fixtures.twcc_feedbacks()
+      expected_feedbacks = Fixtures.twcc_feedbacks_contents()
+
+      encoded_feedbacks
+      |> Enum.zip(expected_feedbacks)
+      |> Enum.each(fn {encoded, expected} ->
+        assert TWCC.decode(encoded) == {:ok, expected}
```

Comment:
Try to put the expected value on the left side. 
`assert {:ok, expected} == TWCC.decode(encoded)`